fetch.js

```diff
@@ -109,9 +110,9 @@
         this._bodyInit = body
         if (typeof body === 'string') {
           this._bodyText = body
-        } else if ('Blob' in self && Blob.prototype.isPrototypeOf(body)) {
+        } else if (blobSupport && Blob.prototype.isPrototypeOf(body)) {
```

Comment:
wasn't sure about this because `blobSupport` actually checks a lot more than just `'Blob' in self`